open-craft/opencraft
# [BB-3966] Fix UX issues

Fix the UX issues mentioned in [this comment](https://tasks.opencraft.com/browse/BB-3618?focusedCommentId=193109&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-193109)

**Related Tickets**
1. [BB-3966](https://tasks.opencraft.com/browse/BB-3966)
2. [Gitlab#723](https://gitlab.com/opencraft/dev/opencraft/-/issues/723)

**Testing Instructions**
1. Checkout to this branch
2. Run ocim frontend and visit http://localhost:3000
3. Verify that all the issues mentioned in [this comment](https://tasks.opencraft.com/browse/BB-3618?focusedCommentId=193109&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-193109) are fixed.
     - [ ] Header, Redeployment Toolbar and Content is aligned properly on left and right
     - [ ] Footer is fixed to the bottom
     - [ ] Navbar active state bleeds off from Manage Course Page (new sidebar)
     - [ ] Fix Redeployment toolbar message. Should say preparing for first deployment.

**Screenshots**
![Screenshot from 2021-04-05 21-19-43](https://user-images.githubusercontent.com/18226212/113594039-c3b88080-9654-11eb-9279-0b25c89033fd.png)


**Reviewers**
